### PR TITLE
Add new card with the number of vulnerabilities (#2467)

### LINKED ISSUE #2467: Add new card with the number of vulnerabilities
Please add a new card with a NUMBER of vulnerabilities to the main vulnerabilities view:

image

### How to do this:
1) go to `VulnerabilityWelcomeView.kt`
2) Create a card inside the following div:

```kotlin
div {
       className = ClassName("col-3 mr-4 mt-5")

// wrap previous code with following:
        div {
            className = ClassName("row")
        }

// add your code:
         div {
            className = ClassName("row")
         }
```

3) Card should have following height:
```kotlin
        style = jso {
            height = 15.rem
        }
```

4) To get the number of Vulnerabilities, you will need to get all vulnerabilities and calculate them. You can do that on the backend or on the frontend. You need to request backend:

```
    val (vulnerabilities, setVulnerabilities) = useState<List<VulnerabilityDto>>(emptyList())

    useRequest {
        val vuln: List<ContestDto> = post(
            url = "$apiUrl/vulnerabilities/get-all-public" OR YOUR CUSTOM ENDPOINT 
            headers = jsonHeaders,
            loadingHandler = ::loadingHandler,
        )
            .unsafeMap { it.decodeFromJsonString() }

        setVulnerabilities(vuln)
    }
```

5) If you wish to create a new backend endpoint, you can check: VulnerabilityController.kt:

```
@RestController
@RequestMapping(path = ["/api/$v1/vulnerabilities"])
class VulnerabilityController(
```

As an example you can use `ContestsStatistics.kt`:

image

### PR DESCRIPTION
**Pull request by JustinB1eber**
### What's done:
* add a new card in welcome view

It closes #2467

### CODE REVIEW

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/VulnerabilityWelcomeView.kt [hunk 1]
```diff
@@ -11,6 +11,7 @@
 
 package com.saveourtool.save.frontend.components.views.welcome
 
+import com.saveourtool.save.entities.vulnerability.VulnerabilityDto
```
[the remark this answers is not in the document]
Comment (nulls) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/VulnerabilityWelcomeView.kt:14]: @JustinB1eber you can run `./gradlew :diktatFix`

**Comment by akuleshov7** [comment]
to run analysis tools, use: `gradlew.bat diktatFix` and `gradlew.bat detekt`

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/VulnerabilityWelcomeView.kt [hunk 2]
```diff
@@ -89,3 +102,29 @@ val vulnWelcomeView: FC<WelcomeProps> = FC { props ->
...
+    div {
+        className = ClassName("card mt-3")
+        div {
+            className = ClassName("mt-5 mb-5")
+            style = jso {
+                height = 15.rem
+            }
+            div {
+                className = ClassName("row justify-content-center")
+                ReactHTML.strong {
```
Comment (nulls) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/VulnerabilityWelcomeView.kt:116]: we use html methods without `ReactHTML.*`, can you add imports for `ReactHtml.strong` and `ReactHtml.h1`?

Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/welcome/VulnerabilityWelcomeView.kt:116]:
```
strong {
```

**Review by akuleshov7 — APPROVED** [review]

**Pull request merged**